ami.tf

```
@@ -3,11 +3,17 @@ locals {
     aws        = "643313122712"
     aws-us-gov = "092348861888"
   }
+
+  archs = {
+    amd64 = "x86_64"
+    arm64 = "arm64"
+  }
+  arch = local.archs[var.autoscaling_configuration["architecture"]]
```

```suggestion
  arch = lookup(local.archs, var.autoscaling_configuration["architecture"], "amd64")
```

`var.autoscaling_configuration` may not be defined